DeAceroDLabs/marketplace-frontend
# Feature/dmp 728 mock  offer on payment

## What is done

[[DMP-728] show offers on payment](https://deacero.atlassian.net/browse/DMP-728)

## How is done

- Added mock Offers data to payment view.
- Created selectedOffer type, this will be used on the context that we will create later.

## How to test

1. Run the project via `npm start`.
2. Click on any product.
3. Click `Agregar al carrito` button.
4. Go to `carrito` then to payment.
5.  On payment file line 50 you can change the offer type to, Express, Estándar or  Similares depending wich you want to see: 
![image](https://user-images.githubusercontent.com/81311646/207966264-e50bad9a-2c8f-4c62-b592-0fffe868c164.png)
